janitor/src/main/scala/com/broadinstitute/dsp/janitor/KubernetesClusterRemover.scala

```diff
@@ -28,8 +28,8 @@ object KubernetesClusterRemover {
     logger: Logger[F],
     ev: Ask[F, TraceId]): CheckRunner[F, KubernetesClusterToRemove] =
     new CheckRunner[F, KubernetesClusterToRemove] {
-      override def appName: String = resourceValidator.appName
-      override def configs = CheckRunnerConfigs(s"remove-kubernetes-clusters", shouldAlert = true)
+      override def appName: String = janitor.appName
+      override def configs = CheckRunnerConfigs(s"remove-kubernetes-clusters", shouldAlert = false)
```

Comment:
Setting `shouldAlert` to `false` since any detections by `Janitor` checkers are not anomaly and shouldn't trigger alerts. 

@Qi77Qi, could you please confirm that this is accurate?